src/style/style.css

```diff
@@ -6343,9 +6343,9 @@ fieldset[disabled] .btn-auth-login.active {
   padding-right: 20px;
 }
 .navbar.navbar-inverse .navbar-nav li a,
-.app-footer h5,
-.app-footer p,
-.app-footer li a {
+.app-footer-old h5,
```

Comment:
i wonder if we should just start to remove these from css instead of renaming, as to start cleaning up this file until it's empty. it will give us more of a vision of the progress we have, as the css file gets smaller, we'll be closer to the cleanup. any specific reason for keeping these here?

Reply:
The idea is to keep them in `style.css` until we are done with the rewrite and then just delete `style.css`. I could've remove these lines, but decided to keep them just in case I need to refer back to old styles during the transition period.

Reply:
okay makes sense!